package.json

```diff
@@ -17,18 +17,23 @@
     "@digitalbazaar/ed25519-verification-key-2020": "^2.1.1",
     "base58-universal": "^1.0.0",
     "ed25519-signature-2020-context": "^1.0.1",
+    "ed25519-signature-2018-context": "^1.1.0",
```

Comment:
It seems to me that it would be appropriate to use `security-document-loader` here.  However I see that it doesn't have the 2018 context:

https://github.com/digitalbazaar/security-document-loader/blob/main/package.json

@dmitrizagidulin thoughts about adding 2018 context to `security-document-loader` to enable its use here?

Reply:
Waiting to hear @dmitrizagidulin's thoughts on this before updating since in the issue it says to use jsonld-document-loader.

Reply:
@mattcollier sure, we can do that. (add the 2018 context to security-document-loader, and use it here for the tests.

Reply:
Updated here 6f73cb0